packages/crud/src/vaadin-crud.js

```diff
@@ -18,6 +18,24 @@ import { MediaQueryController } from '@vaadin/component-base/src/media-query-con
 import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+/**
+ * @private
+ * A to manage slotted button.
+ */
+class ButtonController extends SlotController {
+  constructor(host, type, theme, shouldSetId) {
+    super(host, `${type}-button`, 'vaadin-button', {
+      initializer: (button) => {
+        button.setAttribute('theme', theme);
+
+        if (shouldSetId) {
+          button.id = type;
+        }
```

Comment:
IMO, we should remove `id` attribute from "Save", "Cancel" and "Delete" buttons as these are using very common strings, and there is a risk of conflicts. I will create a separate PR to do that.

Reply:
Submitted #4960 to address this, will rebase this PR once that one is merged.